packages/terra-form-select/src/single/Menu.jsx

```diff
@@ -260,7 +261,21 @@ class Menu extends React.Component {
     }
 
     if (select) {
-      select.focus();
+      if (!SharedUtil.isSafari()) {
+        select.focus();
+      } else {
+        /**
+         * Shifting focus back to the select specifically
+         * when VoiceOver is on will sometimes trigger VO to shift focus
+         * randomly to the root of document or the skip to main link
+         * instead of the select and then break VoiceOver usage when navigating the
+         * select options on subsequent opening of select
+         * Refocusing on select seems to seems to mitigate this VO bug.
+         */
+        setTimeout(() => {
+          select.focus();
+        }, 300);
```

Comment:
This PR was logged to make jaws to announce selected instead of checked. how is time out helping Jaws to pick the selection state of options ..??

Reply:
This was done to fix the irregular behaviour of Safari browser with focus shifting back to select. SR was sometimes announcing when the item gets selected and sometimes it wasn't.